### ISSUE TITLE
Bug: Dark high contrast theme has yellow text on a white background

### ISSUE THREAD

**Issue by choppergirl** [files-community/Files#17730]
### Description

The default view in Windows High Contrast Theme is unreadable to a Blind Person.  Blind people can not read text against a white background, it's like staring into a florescent lamp or into the sun for us.

However, there is a work around, if you change the settings to Dark Theme internally inside Files, it will change the theme for it's window to dark... however, it will lose the Theme settings and revert.

I can't even remember where the Configuration Gear Settings Button is any more...

The Title Bar to "grab" it is missing, or overwritten with Tabs

Compare Windows 10 File Manager on Left, to "Files" File Explorer on Right:

![Image](https://github.com/user-attachments/assets/590b61d9-d11c-4d97-9748-9e1032f5bacf)

I don't suspect this is the "polished" look you were going for....   for Files

### Steps To Reproduce

1. Install Windows 10
2. Switch to Windows High Contrast Theme (dark)
3. Install Files
4. Open Files

### Files Version

Latest

### Windows Version

10

### Log File

I don't even know where the menu item is to create a log file.  This is a nightmare to find even with the help page
https://files.community/docs/getting-started/faq#:~:text=How%20to%20Locate%20the%20Log%20File

I had to open command window and type "dir" and then paste the directory string for those variables to get filled in and me know where the actual directory was,and then you can't even get into the directory because it's set to Hidden

Maybe don't make requiring this log file to be uploaded as a requirement, to submit a bug report, if it's so impossible to do, because it's taking me a full 20 minutes just trying to figure out how to do.. this... which isn't even germaine to the bug being submitted...

Just saying.

I finally was able to do it by copying the file path from a command window, and pasting it into Files itself to go right into the directory bypassing the Hidden flags on upper level directories

[debug.log](https://github.com/user-attachments/files/22703798/debug.log)

**Comment by Josh65-2201** [comment]
Thanks for the report, the log file being more difficult to get to is just for this issue as it not normally hard to click a 3 buttons and open the log files location. Please also don't put personal sites/advertising links in the issue

> However, there is a work around, if you change the settings to Dark Theme internally inside Files, it will change the theme for it's window to dark... however, it will lose the Theme settings and revert.

Do you mean setting the backdrop to none? As that what you said worked in your previous bug report.

> I can't even remember where the Configuration Gear Settings Button is any more...

It's the bottom left of the window

> The Title Bar to "grab" it is missing, or overwritten with Tabs

The grab point on the title bar is blue where the tabs are.

**Comment by yair100** [comment]
Can you please clarify which high contrast theme you're using? All four of the built-in ones are working properly on my end.

**Comment by choppergirl** [comment]
We determined earlier, it seems to have something to do with the Mica-Alt background being set by default...  after a default install.

![Image](https://github.com/user-attachments/assets/245b1213-f568-473b-b63a-21fb9a46874b)

If you change the Backdrop to "None" it works as expected, as pictured above (until I clicked the X close box to save the new setting Mica-Alt from the Pull down menu).

Mica-Alt and Mica Backdrop seems to break it, but None, Acrylic and Thin Acrylic Backdrops are fine... perhaps because they are not displayed or something.

~

I know the High Contrast theme looks very Dayglo, wonky Windows 3.1ish, and even Fisher Price like, and you are probably wondering why in the world would anyone want to rock it, but it's what a person has to use who is visually impaired to be able to see anything at all.   I could not read a thing on a white background, no matter the color...  it was like you trying to read words written on black sharpie marker on a florescent bulb.   Blinding and the light washed out the words.   To look in a book with black text on white paper, all I saw was a blank white page.   Sounds cool, but it really was not.  Like looking out a frosted car window on a winter day.  You can see out, but  you can't.

However, using a High contrast theme with dayglo colors on a solid black background, I could read a computer screen just fine.  In fact, the 5 years I was blind, I probably churned out 300 video edits.

I got surgery on both eyes and now can see just fine, however I grew use to the theme and decided to keep it... as it's fast and easy on the eyes in the dark.

~

No way I have found yet to fix the missing Windows 10 Title Bar Decoration....  

Google Chrome, Discord, and Steam does the same thing monkeying around with the default Windows Interface and breake the Window Title bar and discard it, and it's vexxing as hell not to have a proper Titlebar with the application name on it.  Yeah, you can grab the blue area, but all my other grab Title bar areas are not... blue

![Image](https://github.com/user-attachments/assets/19578064-8012-406b-aaf4-340da8ccf269)

**Comment by Josh65-2201** [comment]
It's not missing as it isn't supported to be there, title bar/window border colors hasn't been used for many programs since Windows 8, on Windows 10 File Explorer hasn't been updated to the new format and that's why it still works for it.

**Comment by yair100** [comment]
> We determined earlier, it seems to have something to do with the Mica-Alt background being set by default... after a default install.

The backdrops shouldn't cause issues with high contrast theme. As I mentioned earlier, I can't reproduce this issue with any of the high contrast themes on Windows 11. Can you please clarify which theme you're using?

**Comment by Josh65-2201** [comment]
@choppergirl Can you answer the above question
